snap/snapcraft.yaml

```
@@ -15,11 +15,11 @@ architectures:
 parts:
   brave:
     plugin: dump
-    source: https://github.com/brave/brave-browser/releases/download/v0.55.22/brave-browser_0.55.22_amd64.deb
+    source: https://github.com/brave/brave-browser/releases/download/v0.56.12/brave-browser_0.56.12_amd64.deb
     source-type: deb
     # Correct path to icon.
     override-pull: |
-      snapcraftctl pull  && rm -rf etc/cron.daily/ &&  rm -rf usr/bin/brave-browser
+      snapcraftctl pull  && rm -rf etc/cron.daily/ &&  rm -rf usr/bin/brave-browser && chmod 755 opt/brave.com/brave/brave-sandbox && sed -i 's|Icon=brave-browser|Icon=/opt/brave.com/brave/product_logo_128\.png|g' usr/share/applications/brave-browser.desktop
```

would it be possible to convert this to multiple lines, please?

```suggestion
      snapcraftctl pull 
      rm -rf etc/cron.daily/
      rm -rf usr/bin/brave-browser
      chmod 755 opt/brave.com/brave/brave-sandbox
      sed -i 's|Icon=brave-browser|Icon=/opt/brave.com/brave/product_logo_128\.png|g' usr/share/applications/brave-browser.desktop
```